dev-client/src/components/buttons/DoneButton.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {useNavigation} from 'terraso-mobile-client/navigation/hooks/useNavigation';
+import {Fab} from 'native-base';
+import {Icon} from 'terraso-mobile-client/components/icons/Icon';
+import {useTranslation} from 'react-i18next';
+import {Box} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type DoneButtobProps = {
```

Comment:
Nitpick: Typo `Buttob` --> `Button`

Reply:
Thanks. Fixed. Next time, please use the GitHub suggest feature — it makes it quicker to implement the change.